public/js/rq_params.js

```diff
@@ -25,7 +25,7 @@ RQ_PARAMS.DISCONNECTED_IMAGE = 'img/background.jpg'
 RQ_PARAMS.ATTR_DELIMIT = ';'
 
 // TODO: Implement use of this in public/js/index.js
-RQ_PARAMS.WIDGET_NAMESPACE = 'rq'
+RQ_PARAMS.WIDGET_NAMESPACE = 'rq_widget'
```

Comment:
I agree this is slightly redundant. RoboQuest widgets are collected into an "rq" jQuery widget namespace, so naming them rq_widget_joystick, for example, instead of "rq_joystick", doesn't really add any additional information.

Reply:
I'll leave it as `rq`